### ISSUE TITLE
Non-resolvable salt isssue

### ISSUE THREAD

**Issue by wirepatch** [Minervis-GmbH/BigBlueButton-Ilias-Plugin#22]
Your plugin requires the GUI `Security Salt` value to contain at least one capital letter and one special character. But the security salt is being provided by a different department and we are not able to change it. Moreover a change would require re-configuration of all client applications spread over our university.

**Comment by xJuvi** [comment]
Hi,
which version did u use?
In my test with version 1.0.12 i can add only a single letter and save it. No capital letters or special chars.
I also reviews the source code in the beta branch for version to. There are also no strict limitations.

Which chararters did your salt contain?

Kind

**Comment by wirepatch** [comment]
Following the installation instructions I just cloned `https://github.com/Minervis-GmbH/BigBlueButton-Ilias-Plugin.git`. So I just assume using the latest version on offer. My salt only contains lowercase letters and digits.

**Comment by wirepatch** [comment]
My only fix so far is changing the underlying database manually e.g.:

```
update rep_robj_xbbb_conf
set svrsalt='adhwdhdhq324e9ujejfwjdjdw'
where id=1;
```

**Comment by wirepatch** [comment]
> Hi, which version did u use? In my test with version 1.0.12 i can add only a single letter and save it. No capital letters or special chars. I also reviews the source code in the beta branch for version to. There are also no strict limitations.

Version is 1.0.15

**Comment by xJuvi** [comment]
I reviewd the code. With  commit https://github.com/Minervis-GmbH/BigBlueButton-Ilias-Plugin/commit/5782a146d7134d561148d3f11e403dc957b3707d (included in v1.0.15) it should be fixed. The option to skip the syntax check of the salt field is set true, so ilias shoudn't check the iput against the password guidelines:
https://github.com/Minervis-GmbH/BigBlueButton-Ilias-Plugin/blob/5782a146d7134d561148d3f11e403dc957b3707d/classes/class.ilBigBlueButtonConfigGUI.php#L93-L95

The problem is, this change is never merged to the main branch, which you had used to install the plugin. I've created PR #24 to resolve your bug.
